plugin.video.manoto/default.py

```diff
@@ -10,21 +9,6 @@
 li.setArt({ 'thumb': icon })
 li.setInfo('video', {'plot': 'Live Stream'})
 
-livePageUrl = "https://www.manototv.com/live"
+liveVideoUrl = "https://dow6lwapg8fa4.cloudfront.net/live_2500.m3u8"
 
-request = urllib2.Request(livePageUrl, headers={'User-Agent' : "Kodi"})
-try:
-    response = urllib2.urlopen(request)
-    html = response.read()
-
-    soup = BeautifulSoup(html)
-    stream = soup.find('source', type='application/x-mpegURL')
-
-    if stream is not None and stream['src'] is not None:
-            xbmc.Player().play(stream['src'], li, False)
-    else:
-        xbmcgui.Dialog().ok(addonname,  __language__(30002), __language__(30010))
-except urllib2.HTTPError, err:
-        xbmcgui.Dialog().ok(addonname,  __language__(30000) % err.code, __language__(30010))
-except:
-        xbmcgui.Dialog().ok(addonname,  __language__(30001), __language__(30010))
+xbmc.Player().play(liveVideoUrl, li, False)
```

Comment:
Calling `xbmc.Player().play(...)` isn't allowed for a plugin.
You have to create a directory listing, which contains one `ListItem`. For that list item you set the path to the above URL.